mon-pix/app/templates/update-expired-password.hbs

```diff
@@ -1,47 +1,58 @@
 {{title pageTitle}}
-<div class="sign-form-page">
-  <div class="sign-form__container">
+<div class="update-expired-password-form-page">
+  <div class="update-expired-password-form__container">
 
     <a href={{this.urlHome}} class="pix-logo__link" title="Lien vers la page d'accueil de Pix">
       <img class="pix-logo__image" src="{{@rootURL}}/images/pix-logo.svg" alt="Pix">
     </a>
 
-    <div class="sign-form__header">
-      <h1 class="sign-form-title">{{@model.username}}</h1>
+    <div class="update-expired-password-form__header">
+      <h1 class="update-expired-password-form-title">Nouveau mot de passe</h1>
 
-      {{#unless this._displaySuccessMessage}}
-        <div class="sign-form-header__instruction sign-form-subtitle">Saisissez votre nouveau mot de passe</div>
+      {{#unless displaySuccessMessage}}
+        <div class="update-expired-password-form-header__instruction update-expired-password-form-subtitle">
+          Vous avez utilisé un mot de passe à usage unique.  <br>
+          Pour continuer, vous devez choisir un nouveau mot de passe.
+        </div>
       {{/unless}}
     </div>
 
-    {{#unless this._displaySuccessMessage}}
-      <form {{action "handleUpdatePassword" on="submit"}} class="sign-form__body">
+    {{#unless displaySuccessMessage}}
```

Comment:
remettre les 'this.' en place pour displaySuccessMessage (améliore la clarté).

Reply:
Done